sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_deserialize.py

```diff
@@ -96,6 +98,10 @@ def from_blob_properties(blob_properties):
     file_props.deleted_time = blob_properties.deleted_time
     file_props.remaining_retention_days = blob_properties.remaining_retention_days
     file_props.content_settings = blob_properties.content_settings
+
+    # TODO: This is definitely not how we want to do this
+    if blob_properties.has_key('encryption_context'):
+        file_props.encryption_context = blob_properties.encryption_context
```

Comment:
Also want to call this one out that it came through in the `blob_properties` (because we forced it to) and then we need to finish the plumbing job